client/src/bookingPortal.jsx

```diff
@@ -21,10 +21,13 @@ class BookingPortal extends React.Component {
     this.state = {
       currentListing: {},
       currentAvailability: [],
+      inputClick: 0,
```

Comment:
Maybe more appropriate to use a boolean here rather than 0/1?

Reply:
haha yea XD lemme fix that!